src/map.tsx

```diff
@@ -264,7 +265,7 @@ const ReactMapboxFactory = ({
         zoom: zoom ? zoom[0] : defaultZoom[0],
         minZoom,
         maxZoom,
-        maxBounds,
+        maxBounds: this.props.maxBounds || maxBounds,
```

Comment:
Can you remove ` || maxBounds` and remove it from the factory parameter (line 197), also destruct `maxBounds` from `this.props` (line 25*)